ui/uihandler.go

```diff
@@ -920,3 +922,18 @@ func setHostAndPort(inputPort string, addr *net.TCPAddr) {
 		urlHostPort = "localhost" + portStr
 	}
 }
+
+// addHTTPS replaces "http://" in url with "https://" or prepends "https://"
+// if url does not contain prefix "http://".
+func addHTTPS(url string) string {
+	if strings.HasPrefix(url, "http://") {
+		log.Infof("replacing http scheme with https for url: %s", url)
+		return strings.Replace(url, "http://", "https://", 1)
```

Comment:
it's not the cheapest/cleanest way to do this (replace is a search and replace, we already know it starts with http://) but ok for now

Reply:
OK. I will trim if the url has a prefix of `http://` and return `https://` + the trimmed url.